### PR TITLE
build(docs): adjust the Makefile docs target

### PR DESCRIPTION
**Pull request by behnazh-w**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -279,8 +279,7 @@ dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-build-epoch.txt:
 # Build the HTML documentation from the package's source.
 DOCS_SOURCE := $(shell find docs/source -type f -name '*.rst')
 .PHONY: docs
-docs: docs/_build/html/index.html
-docs/_build/html/index.html: $(DOCS_SOURCE)
+docs: $(DOCS_SOURCE)
```
[the remark this answers is not in the document]
Comment (behnazh-w) [reply Makefile:282]: > Lgtm. Do you think we should also include the `docs/source/conf.py` file as a prerequisite for the `docs` target?

Yes, good idea.

[the remark this answers is not in the document]
Comment (tromai) [reply Makefile:282]: I think `docs/source/conf.py` would be reasonable too.

File: Makefile [hunk 2]
```diff
@@ -277,10 +277,14 @@ dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-build-epoch.txt:
 	echo $(SOURCE_DATE_EPOCH) > dist/$(PACKAGE_NAME)-$(PACKAGE_VERSION)-build-epoch.txt
 
 # Build the HTML documentation from the package's source.
-DOCS_SOURCE := $(shell find docs/source -type f -name '*.rst')
+DOCS_SOURCE := $(shell find docs/source -type f -name '*.rst' -or -name '*.py' -or \
```
Comment (nathanwn) [inline Makefile:280]: Hmm, how about we use this instead:
```make
DOCS_SOURCE := $(shell git ls-files docs/source)
```
This makes doc sources all files under `docs/sources`, and we don't have to maintain a list of extensions in the `find` command, which can be pretty error-prone and hard to maintain. I think the current list is missing `*.html` for the sample HTML reports and `*.svg` for some diagrams, for example.

Comment (behnazh-w) [reply Makefile]: @nathanwn Thanks for the suggestion. See commit [786e599](https://github.com/oracle/macaron/pull/370/commits/786e599bb509590c9311b06d4c76347ea6c541f9).

**Review by nathanwn — APPROVED** [review]

**Pull request merged**